astro/src/content/docs/get-started/start-here/step-4.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Step 4 - Role-Based Access Control"
+description: Learn how to use FusionAuth by using an example application that has a protected page defined.
+section: get started
+subcategory: start here
+navOrder: 0
+codeRoot: https://raw.githubusercontent.com/fusionauth/fusionauth-example-get-started/main
+---
+
+<div className="border border-green-700 px-3 rounded-md">
+  <i className="fa-duotone fa-stopwatch fa-2x mr-2 text-green-700 align-middle"/> Estimated time to complete this step: **5 minutes**
+</div>
+
+## Setting up roles
+
+The [Kickstart](/docs/get-started/download-and-install/development/kickstart) from step one created two roles for our Changebank application. These are `admin` and `user`. It assigned these roles to the two users it created as well. The user `admin@example.com` was granted the `admin` role and the user `richard@example.com` was granted the `user` role.
```

Comment:
This is where the RemoteCode being able to pull from Kickstart might be useful.

I don't see the `user` role being created in https://github.com/FusionAuth/fusionauth-example-kickstart/blob/main/get-started/kickstart.json or granted to the richard@example.com user.

Am I looking at the wrong kickstart file?

Reply:
Yeah. It's on a branch. And I'm purposely not pulling in the Kickstart file because it's crazy long and will just bloat this doc. I'm trying to provide information here without overwhelming the reader.